File: src/bin/some_script.pl
#!/usr/bin/perl

use strict;
use warnings;

use lib '/opt/IAS/lib/perl5';

use FindBin qw($RealBin);
use lib "$RealBin/../lib/perl5";

use IAS::HelloRepoLayout;

IAS::HelloRepoLayout::hello();

# We can also do OO easily:

my $app = IAS::HelloRepoLayout->new();
$app->hello();

File: src/bin/some_script.rb
#!/usr/bin/ruby

realbin = $0
if File.lstat(realbin).symlink?
  realbin = File.readlink($0)
end

$LOAD_PATH.unshift('/opt/IAS/lib/ruby2')
$LOAD_PATH.unshift(File.expand_path('../lib/ruby2/', File.dirname(realbin)))

require 'ias/hello_repo_layout'

app = IAS::HelloRepoLayout.new

app.hello
